05_syntax_lab/05.py

```diff
@@ -1,5 +1,22 @@
-"""
-Write a program that randomizes integers in a loop
-until it finds a number that is divisable by: 7, 13 and 15
-"""
-
+from random import randint
+
+print "Exercise # 5"
+
+
+while True:
+    num = randint(1, 1000000)
+    print "Number is %d" % (num)
+    if num % 7 == 0:
+        if num % 13 == 0:
+            if num % 15 == 0:
+                print "Number %d is divided by 7,13,15" % (num)
+                break
+            else:
+                print "Number %d is NOT divided by 15" % (num)
+        else:
+                print "Number %d is NOT divided by 13" % (num)
+    else:
+                print "Number %d is NOT divided by 7" % (num)
+                
+        
+

```

Comment:
I can make it within the same if statement using and :)
if ((num % 7 == 0) && (num % 13 == 0)  && (num % 15 == 0)):

or using elif :

while True:
    num = randint(1, 1000000)
    print "Number is %d" % (num)
    if num % 7 > 0:
        continue
    elif num % 13 > 0:
        continue
    elif num % 15 > 0:
        continue
    else:
        print "Number %d is divided by 7,13,15" % (num)
        break